ground/gcs/src/plugins/boards_dtf/dtfcconfiguration.ui

```diff
@@ -757,7 +757,6 @@
  <resources>
   <include location="dtfuhf.qrc"/>
   <include location="../coreplugin/core.qrc"/>
-  <include location="../coreplugin/core.qrc"/>
```

Comment:
I've stopped bothering with fixing this. It doesn't affect UIC so may as well not fight it. It's fine to leave it removed here though.